brave/brave-ios
# Fix #3098: Use different icon for insecure sites

## Summary of Changes


This pull request fixes #3098

## Submitter Checklist:


## Test Plan:

- Visit `https` link, lock should be the same
- Visit `http` link, should show warning symbol instead of lock now

## Screenshots:

| Secure (Unchanged) | Insecure |
| --- | --- |
| ![Simulator Screen Shot - iPhone 12 Pro - 2020-12-02 at 14 47 56](https://user-images.githubusercontent.com/529104/100923806-becca000-34ad-11eb-83d1-63d5fdfbdce9.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2020-12-02 at 14 47 57](https://user-images.githubusercontent.com/529104/100923816-c1c79080-34ad-11eb-9294-34912d0948f7.png) |

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
